File: Material de Apoio/002 - Introducao à Arquitetura do Spark - Parte 01/FlorentinaCount.scala
import org.apache.spark.sql.SparkSession

val spark = SparkSession.builder.getOrCreate()

val florentina_path = "/FileStore/tables/florentina.txt"

val florentinaRDD = spark.sparkContext.textFile(florentina_path)

val wordFlorentinaRDD = florentinaRDD.flatMap(x => x.split(" "))

val filterFlorentinaRDD = wordFlorentinaRDD.filter(x =>  x.contains("Florentina"))

filterFlorentinaRDD.count()

File: Material de Apoio/002 - Introducao à Arquitetura do Spark - Parte 01/FlorentinaCount.py
from pyspark.sql import SparkSession

spark = SparkSession.builder.getOrCreate()

path_florentina = "D:\\PyCharm\\Blueshift-DatabricksSpark\\Material de Apoio\\florentina.txt"

florentinaRDD = spark.sparkContext.textFile(path_florentina)

wordFlorentinaRDD = florentinaRDD.flatMap(lambda x: x.split(" "))

filterFlorentinaRDD = wordFlorentinaRDD.filter(lambda x: 'Florentina' in x)

filterFlorentinaRDD.count()